src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -233,12 +235,15 @@ const WalletTable = ({
       <WalletContentHeader>
         <Th sx={{ textAlign: "start !important" }}>
           <Flex justifyContent="space-between" px={{ base: 2.5, md: 0 }}>
+            {/* Displayed on mobile only */}
             <Text
               hideFrom="lg"
               lineHeight={1.6}
               fontSize="md"
               color="primary.base"
               textTransform="uppercase"
+              cursor="pointer"
+              onClick={onOpen}
             >
```

Comment:
Just curious, is there any reason we don't want to use a button here? Seems more appropriate from an a11y standpoint

Reply:
Agreed. Now, we are basically turning this component into a button.

Reply:
@wackerow It was defined in the design that should behave like a button but without the button styles. I think I can add the `as={button}` prop to improve a11y

Reply:
Okay cool.. yeah that raising a small design system question... we have `variant="link"` for our buttons, but this will bold and underline the text (inconsistent with what we want here)... @nloureiro Should we have another variant that is even more simplified for buttons like this? Or should we convert this one to use the `link` variant styling?

Reply:
@wackerow do you think that could be a blocker for this PR or could be address later?

Reply:
Nope, not at all! We can address design system stuff separately, this works